Infrastructure-Scripts/Remove-AzSqlIpException.ps1

```diff
@@ -0,0 +1,51 @@
+<#
+    .SYNOPSIS
+    Remove the whitelisted IP from the firewall rule name
```

Comment:
Can this be changed to what the script is doing? not removing from a firewall rule name